src/components/Text/index.tsx

```diff
@@ -4,23 +4,29 @@ import { localeOptions } from 'primereact/api';
 
 import Context from './context';
 
-import { ComponentProps } from './Text.types';
+import { ComponentProps, HookParams } from './Text.types';
 
-const Text: ComponentProps = ({ id, lang, params, prefix, children }) => {
+export const useText = ({ id, lang, params, prefix, text: textInput }: HookParams) => {
```

Comment:
Rollup gives warnings "The following entry modules are using named and default exports together".
Best is to move this hook in the hooks folder.

Reply:
Its moved